src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -398,13 +352,15 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       applyThemesOnElement(this, this.hass.themes, this._config.theme);
     }
 
+    this._updateProgressBar();
+
     if (
       !this._progressInterval &&
-      this._showProgressBar &&
+      this._showProgressBar !== false &&
```

Comment:
Why check this to false if its a boolean

Reply:
oops old stuff.